packages/docs/src/pages/color-modes.mdx

```diff
@@ -148,3 +148,25 @@ If you do not have a color mode named `dark` or `light`, this will have no effec
 }
 ```
 
+### Disable persisting color mode on `localStorage`
+
+In some cases, it is not desirable to have the color mode set as a `localStorage` value. 
+The common use-case being that disconnected slices of your app use different color 
+modes on the same domain.
```

Comment:
I'd prefer not to get into the *why* someone would want to disable `localStorage` to avoid any sort of judgemental sounding language, if that makes sense
```suggestion
To disable `localStorage` add the `useLocalStorage: false` flag to your theme.
```